unabridged-works-volume-3/original/11-flesh-blood-mystery.adoc

```diff
@@ -308,6 +318,8 @@ with me in this respect,
 since it pleased the Lord and Father of mercies to
 reveal the mystery of himself and of his Son in me.
 
+====
+
 In the first place, I freely confess, that I do own and acknowledge, as in God`'s sight,
```

Comment:
So I wasn't sure if I should start the number group here and italicize that whole phrase, or just the word "first", but this seemed to look the best to me, even though it's technically part of the numbered group below it. But you decide!

Reply:
Hmm... i'm not sure either. i might play with it after i merge a bit, but i think i'll leave it how you have it here for now.